### ISSUE TITLE
Scalaz: Adopt stronger branding & create repository for scalaz.org / scalaz.github.io

### ISSUE THREAD

**Issue by jdegoes** [scalaz/scalaz#1418]
![scalaz](https://user-images.githubusercontent.com/156745/27654636-ea297bea-5bff-11e7-9b37-1bb8687027b0.png)

[Scalaz.svg.zip](https://github.com/scalaz/scalaz/files/1109888/Scalaz.svg.zip)

**Comment by fommil** [comment]
Nice. What's the licence on the logo?

**Comment by jdegoes** [comment]
What do you want it to be? I bribed my brother to create it for a package of caramel cookie waffles. 😉

**Comment by fommil** [comment]
CC-BY-SA is always my preference. But I suspect scalaz contributors may prefer CC-BY (the pushover variant)

**Comment by tonymorris** [comment]
That logo looks pretty awesome John.

@fommil Me, you, bottle of scotch. Let's push and see.

**Comment by tonymorris** [comment]
Also, I completely agree with the idea of stronger branding, sorting out domain names, getting it organised, etc.

**Comment by jdegoes** [comment]
CC BY-SA 4.0. 👍 

I see that both scalaz.org and scalaz.com are registered and currently point to this Github repository. All we need to do is create a `scalaz.github.io` repository on the `scalaz` organization, then setup the domain names to point to Github Pages servers, and finally add a small file to the repository, and then we'll have the basics for a static website. 

The logo can also be used on the README for the repository (/repositories).

Anyone know who controls scalaz.org, scalaz.com?

**Comment by relrod** [comment]
@jdegoes This looks amazing!

They're both registered to Jason Zaugg (@retronym):

https://da.gd/w/scalaz.org
https://da.gd/w/scalaz.com

I'd be happy to help throw together a landing page with Hakyll or something.

**Comment by jdegoes** [comment]
@relrod If no one minds, perhaps we should use Jekyll so we can have Github Pages build it. Otherwise we need to have a separate deployment process and require Haskell tooling.

@retronym You doing anything with those domain names?

**Comment by retronym** [comment]
AFAICR, I transferred the domains to @tonymorris (see https://github.com/scalaz/scalaz/issues/916)

It appears that the contact details in the domain records wasn't updated.

**Comment by stefanobaghino** [comment]
Resembles a lot [the insignia NASA adopted around the '80s](https://en.wikipedia.org/wiki/NASA_insignia).

![NASA Insignia](https://upload.wikimedia.org/wikipedia/commons/a/a3/NASA_Worm_logo.svg)

**Comment by jdegoes** [comment]
Thanks, @retronym. @tonymorris, if you can transfer them to me or at me as an admin, I'm happy to do the setup.

**Comment by relrod** [comment]
> @relrod If no one minds, perhaps we should use Jekyll so we can have Github Pages build it. Otherwise we need to have a separate deployment process and require Haskell tooling.

@jdegoes I'm less familiar with Jekyll and don't have Ruby set up on my machine, but sure... Either way. :smile:

**Comment by aloiscochard** [comment]
I don't see a massive advantage at using Jekyll + GitHub Pages.

Hakyll is great and it's not a big deal to do the deployment locally, also if you really care about the pipeline you can get it to be built by Travis: http://timbaumann.info/posts/2013-08-04-hakyll-github-and-travis.html

Feel free to discard my comment as I don't plan to spend time on the website, so do what is the best for you. It would be just be a shame to favor Jekyll over Hakyll just for integration reasons.

**Comment by rabbitonweb** [comment]
How we moved that forward some how? I'm happy to contribute to scalaz for start with this in mind https://twitter.com/mjpt777/status/898143122617761792

**Comment by fommil** [comment]
@rabbitonweb note that I rewrote https://leanpub.com/fp-scala-mortals to target Scalaz instead of cats, so we will have a textbook for beginners too.

**Comment by jdegoes** [comment]
Only an admin has the ability to create a new repository `scalaz.github.io`. /cc @tonymorris

**Comment by puffnfresh** [comment]
@jdegoes there's already a gh-pages branch: http://scalaz.github.io/scalaz/#scaladoc - is that enough?

**Comment by jdegoes** [comment]
@puffnfresh That should be enough to start. As I recall, domain names cannot be mapped to gh-pages (project docs).

Are the existing contents important at all?

**Comment by puffnfresh** [comment]
@jdegoes the links to Scaladoc have been requested a lot. I think they are valuable.

**Comment by fommil** [comment]
Why not just link out to http://www.javadoc.io or include instructions for browsing them offline with ensime? C-c C-v D

**Comment by S11001001** [comment]
@fommil can this be used for [`apiURL`](http://www.scala-sbt.org/0.13/docs/Howto-Scaladoc.html#Define+the+location+of+API+documentation+for+a+library) with generated URLs being right?

**Comment by fommil** [comment]
@S11001001 huh, I never knew about that feature. We should probably support that in ensime if it's in the pom/ivy files. I don't see why it wouldn't work.

**Comment by jdegoes** [comment]
@puffnfresh Makes sense, I think the links can be located on the website in a "docs" section.

Do folks have examples of OSS library websites they find quite useful and clearly laid out?

**Comment by fommil** [comment]
It's supposed to be a scala logo, but I'm calling Z on the scala world "scala center spree" logo:

![](https://storage.googleapis.com/scalaworld-www/images/scala_center.jpg)

**Comment by fommil** [comment]
that was just me joking, but it got me thinking... if adopting stronger branding, Scalaz should probably have a logo with just the Z in it. Maybe starting from the scala logo isn't a bad idea, but it needs to be sharper, maybe a strip of red hot metal newly bent into an elongated Z.

**Comment by aloiscochard** [comment]
It's like with coherence: there is only one proper way to draw a Z!

https://www.youtube.com/watch?v=XumDitAgl9c

**Comment by jdegoes** [comment]
I like this idea. Now does anyone of the talent to create such a thing? 😆

**Comment by fommil** [comment]
could somebody please change the scalaz org logo to a Z? Even a faked github randomised logo in the shape of a Z would be better than the default https://github.com/scalaz

**Comment by jdegoes** [comment]
@fommil I think @puffnfresh is an Admin and can do this. I would if I had appropriate permissions but I only have admin on the Scalaz repo (like you, btw—welcome to Scalaz!).

**Comment by puffnfresh** [comment]
Updated
